docs/manual_doc/Functions_and_Operators/Comparison_Functions_and_Operators.md

```diff
@@ -0,0 +1,96 @@
+| Name  | Description | 
+|:----|:----:|
+|   ==     |   Equal operator     | 
+|   >     |   Greater than operator     | 
+|   >=     |   Greater than or equal operator     | 
+|   <     |   Less than operator    | 
+|   <=     |   Less than or equal operator    | 
+|   !=    |   Not equal operator    | 
+
+
+Comparison operations result in a value of _true_ and _false_.
+
+* ==
+
+Equal. String comparisons are case-sensitive. Values of different type are not equal.
+
+```
+nebula> YIELD 'A' == 'a';
+==============
+| ("A"=="a") |
+==============
+|      false |
+--------------
+
```

Comment:
The console output is now left aligned
Many of the following are similarly handled